src/store/breakoutRoomsStore.js

```diff
@@ -32,68 +32,82 @@ import {
 	reorganizeAttendees,
 } from '../services/breakoutRoomsService.js'
 import { showError } from '@nextcloud/dialogs'
-import { set } from 'vue'
+import Vue from 'vue'
 import { emit } from '@nextcloud/event-bus'
 
 const state = {
-	breakoutRoomsReferences: {},
+	breakoutRooms: {},
 }
 
 const getters = {
-	breakoutRoomsReferences: (state) => (token) => {
-		if (!state.breakoutRoomsReferences?.[token]) {
+	breakoutRooms: (state) => (token) => {
+		if (!state.breakoutRooms?.[token]) {
```

Comment:
```suggestion
		if (!state.breakoutRooms.[token]) {
```
Shouldn't be undefined